provisioning/cicd-install

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+#
+# Installs and Configures the CI/CD Server
+#
+# * Java JDK
+# * Apache Maven
+# * Jenkins CI Server
+# * Sonatype Nexus Artifact Repository
+#
+
+set -e
+
+#
+# Constants
+#
+
+JAVA_VERSION=7u79
+JAVA_VERSION_BUILD=b15
+CICD_SETUP_DIR=/tmp/cicd-setup
+CLOUD_ENV=0
+NEXUS_VERSION=2.11.3-01
+MAVEN_VERSION=3.3.3
+
+SCRIPT_BASE_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+# Set Trap
+trap clean_up EXIT
+
+function clean_up() {
+
+  if [ $? -eq 0 ]; then
+	echo
+	echo "====================================================="  
+	echo "= CICD Server Provisioning Completed Successfully!  ="
+	echo "====================================================="  
+	echo
+  else
+  	echo
+  	echo "====================================================="  
+  	echo "= CICD Server Provisioning Failed!                  ="
+  	echo "====================================================="  
+  	echo
+fi
+
+  # Cleanup
+  rm -rf ${CICD_SETUP_DIR}
+
+}
+
+
+#
+# Function to install prerequisite packages
+#
+
+function install_prereqs() {
+	yum install -y wget git
+}
+
+#
+# Function to install the Java JDK
+#
+
+function install_java_jdk() {
+
+  wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" -O "${CICD_SETUP_DIR}/jdk-${JAVA_VERSION}-linux-x64.rpm" "http://download.oracle.com/otn-pub/java/jdk/${JAVA_VERSION}-${JAVA_VERSION_BUILD}/jdk-${JAVA_VERSION}-linux-x64.rpm"
```

Comment:
@sabre1041 is there a reason you're manually downloading an RPM? We have the oracle JDK in software collections.